Documentation/local-timestepping.rst

```diff
@@ -88,9 +88,12 @@ SeisSol also supports the automatic merging of clusters.
 For this, you need to set *LtsAutoMergeClusters* to one.
 The parameter *LtsAllowedRelativePerformanceLossAutoMerge* controls the allowed relative performance loss when merging clusters compared
 to the baseline cost.
-Currently, there are two different types of computing the baseline cost.
-The standard is :code:`LtsAutoMergeCostBaseline = 'bestWiggleFactor'`, which computes the optimal wiggle factor without merging and uses its cost as the baseline.
+There are two different types of computing the baseline cost.
```

Comment:
Sounds like there are 2 options, but one option is anyway useless, so why keeping it?

Reply:
IMO It is not useless. If you have a simulation (baseline) and want to run it with as few clusters as possible then maxWiggleFactor makes sense. It's not that much effort to support it, as it's <50 lines of code, including parameter parsing.

Reply:
If it turns out that no one uses it, we can remove it later